keps/sig-cloud-provider/991-cloud-controller-migration/README.md

```diff
@@ -163,29 +169,50 @@ type ControllerLeaderConfiguration struct {
 	// E.g. service-controller, route-controller, cloud-node-controller, etc
 	Name string `json:"name"`
 
-	// Component is the name of the component in which the controller should be running.
+	// Component is the name of the component in which the controller will be running.
 	// E.g. kube-controller-manager, cloud-controller-manager, etc
 	Component string `json:"component"`
 }
 ```
+#### Default LeaderMigrationConfiguration
+
+The `controller-manager` will provide `kube-controller-manager` and `cloud-controller-manager`
+each a default `LeaderMigrationConfiguration` that represents the situation where the controller manager is running with
+default assignments of controllers and lock type selection.
+
+The default values must be only used when no configuration file is specified. If a custom configuration file is
```

Comment:
Can we specify what the default controllers/values are in the default configuration?

Reply:
I believe not at this stage of designing. The important part is that we support a default value.

Reply:
+1, but if we don't define the values here, we do need to define the flags and resulting behaviors. Eg which controllers are being moved, and a walk through of the Controllers XYZ will run in KCM version N with flag P on the KCM. In version N+1 and the flag P on the CCM (and cloud-provider set to external on the KCM), Controllers XYZ will be running on the CCM. During an HA upgrade from N to N+1 the Controllers XYZ may be running on either the KCM or the CCM, but never both. It is not true that because the Controllers were running on the CCM at any point during the upgrade, that they cannot run on the KCM after that, during the upgrade.

Reply:
I think there the explanation at the beginning of the example should be sufficient. Will polish a bit further.

Reply:
Add specific flags in the example. The walk-though should be follow-able now